Cargo.toml

```diff
@@ -33,14 +33,14 @@ wat = "1.0.49"
 
 wit-bindgen-core = { path = 'crates/bindgen-core', version = '0.2.0' }
 wit-bindgen-gen-guest-c = { path = 'crates/gen-guest-c', version = '0.2.0' }
-wit-bindgen-gen-guest-rust = { path = "crates/gen-guest-rust", version = "0.2.0" }
+wit-bindgen-gen-guest-rust = { path = "crates/gen-guest-rust", version = "0.2.0", default-features = false }
 wit-bindgen-gen-guest-teavm-java = { path = 'crates/gen-guest-teavm-java', version = '0.2.0' }
 wit-bindgen-gen-host-js = { path = 'crates/gen-host-js', version = '0.2.0' }
 wit-bindgen-gen-host-wasmtime-py = { path = 'crates/gen-host-wasmtime-py', version = '0.2.0' }
 wit-bindgen-gen-host-wasmtime-rust = { path = 'crates/gen-host-wasmtime-rust', version = '0.2.0' }
 wit-bindgen-gen-markdown = { path = 'crates/gen-markdown', version = '0.2.0' }
-wit-bindgen-gen-rust-lib = { path = 'crates/gen-rust-lib', version = '0.2.0' }
-wit-bindgen-guest-rust = { path = 'crates/guest-rust', version = '0.2.0' }
+wit-bindgen-gen-rust-lib = { path = 'crates/gen-rust-lib', version = '0.2.0', default-features = false }
+wit-bindgen-guest-rust = { path = 'crates/guest-rust', version = '0.2.0', default-features = false }
```

Comment:
With my suggestion below it will remove the need to say `default-features = false` on the `gen-guest-rust` and `gen-rust-lib` crates, and with those changes is this still necessary?

I'll admit that I do wish that `workspace = true, default-features = false` would indeed turn off default features but alas. If that's the point of conflict you're running into I think it might be reasonable to, for that particular dependency line, avoid `workspace = false`

Reply:
Yes, that change got rid of all the default-features changes in this PR.